research-hub-web/src/app/components/articles/articles.component.spec.ts

```diff
@@ -76,4 +315,28 @@ describe('ArticlesComponent', () => {
     expect(de.nativeElement.innerHTML).toEqual('Article Collection');
   });
 
+  describe('When a truthy url slug is present', () => {
```

Comment:
Might be better to rename the test like "When a slug is present"? Truthy sounds a bit confusing IMO? :thinking:

Reply:
Done